docs/permissions/policies.md

```diff
@@ -10,243 +10,282 @@ Each Role you assign to  user or user group consists of Policies which define, w
-|---------------|----------------------|-----------------------------|----------------------|
-| `checkout` | `view` | access checkout ||
-|| `create` | create new checkout, for example, after workflow fails to complete |
-|| `update` | change currency, quantity |
-|| `delete` | delete checkout, for example, after workflow completes successfully |
-
-### Orders [[% include 'snippets/commerce_badge.md' %]]
-
-| Module| Function| Effect| Possible Limitations |
-|---------------|----------------------|-----------------------------|----------------------|
-| `order` | `create` | create an order |[OrderOwner](limitation_reference.md#order-owner-limitation)|
-|| `view` |view orders|[OrderOwner](limitation_reference.md#order-owner-limitation)|
-|| `update` | change status of an order |[OrderOwner](limitation_reference.md#order-owner-limitation)|
-|| `cancel` | cancel an order |[OrderOwner](limitation_reference.md#order-owner-limitation)|
-
-### Shipping methods [[% include 'snippets/commerce_badge.md' %]]
-
-| Module| Function| Effect| Possible Limitations |
-|---------------|----------------------|-----------------------------|----------------------|
-| `shipping_method` | `create` | create a shipping method |
-|| `view` |view shipping methods|
-|| `update` | modify a shipping method |
-|| `delete` | delete a shipping method |
-
-### Shipments [[% include 'snippets/commerce_badge.md' %]]
-
-| Module| Function| Effect| Possible Limitations |
-|---------------|----------------------|-----------------------------|----------------------|
-| `shipment` | `create` | create a shipment |[ShipmentOwner](limitation_reference.md#shipment-owner-limitation)|
-|| `view` |view shipments|[ShipmentOwner](limitation_reference.md#shipment-owner-limitation)|
-|| `update` | change status of a shipment |[ShipmentOwner](limitation_reference.md#shipment-owner-limitation)|
-|| `delete` | delete a shipment |[ShipmentOwner](limitation_reference.md#shipment-owner-limitation)|
-
-### Payment methods [[% include 'snippets/commerce_badge.md' %]]
-
-| Module| Function| Effect| Possible Limitations |
-|---------------|----------------------|-----------------------------|----------------------|
-| `payment_method` | `create` | create a payment method |
-|| `view` |view payment methods|
-|| `edit` | modify a payment method |
-|| `delete` | delete a payment method |
-
-### Payments [[% include 'snippets/commerce_badge.md' %]]
-
-| Module| Function| Effect| Possible Limitations |
-|---------------|----------------------|-----------------------------|----------------------|
-| `payment` | `create` | create a payment |[PaymentOwner](limitation_reference.md#paymentowner-limitation)|
-|| `view` |view payments|[PaymentOwner](limitation_reference.md#paymentowner-limitation)|
-|| `edit` | modify a payment |[PaymentOwner](limitation_reference.md#paymentowner-limitation)|
-|| `delete` | delete a payment |[PaymentOwner](limitation_reference.md#paymentowner-limitation)|
+### All
```

Comment:
Consider turning the `All` module and table into a Tip with a statement that "you can select `All modules` to grant users access to all modules, and `All functions` to grant them access to all functions within a module."

A Tip would raise attention and allow you to get rid of the additionaal rows with asterisk from all the tables below.

Reply:
`*` has been removed. The tip has been added. But the section "All" is still there, just renamed "Access to all functions"

Authors approval after those last changes.

I resolve this outdated conversation.